launcher/src/backend/ServiceManager.js

```diff
@@ -835,12 +800,12 @@ export class ServiceManager {
 
   async createKeystores(services) {
     for (const service of services) {
-      if (service.service === "NimbusValidatorService" || (service.service === "NimbusBeaconService" && service.configVersion < 2)) {
+      if (service.service.includes("Nimbus")) {
```

Comment:
guys if you have merge conflicts don't just push anything... if you're not sure, then ask the one who wrote this and don't just do something 😅

Reply:
got it sir:)